src/diagnosticMessages.json

```diff
@@ -46,6 +46,7 @@
   "Property '{0}' is always assigned before being used.": 233,
   "Expression does not compile to a value at runtime.": 234,
   "Only variables, functions and enums become WebAssembly module exports.": 235,
+  "Circular inheritance": 236,
```

Comment:
It's better to use TypeScript's errors if possible; i.e. add and use TS2506 for this
`"'{0}' is referenced directly or indirectly in its own base expression": 2506`

Reply:
Thanks, done.